### ISSUE TITLE
Traceback errors on export

### ISSUE THREAD

**Issue by ozpolonski** [flybywiresim/glTF-Blender-IO-MSFS#6]
**Describe the bug**
Trying to export a modified Asobo aircraft is giving me traceback errors:

![Greenshot(002986)-20211122](https://user-images.githubusercontent.com/74175203/142888539-94bd8c26-da44-4c40-b89a-599a7056c77b.png)

I've recently reinstalled Blender 2.92 with Windows 10, no other MSFS stuff active (getting activation issues with MSFSToolkit too). I deleted Blender originally due to this issue too, hoping it would resolve on reinstall.

**To Reproduce**
As always:
1. Select all items to export
2. choose Export
![Greenshot(002989)-20211122](https://user-images.githubusercontent.com/74175203/142889116-9e8e7843-1cf6-463f-879f-0254a0c89fdb.png)
3. Select the file to export
4. Get the error posted above 

**Expected behavior**
As always, a black loading square would appear, and the .gltf and .bin files would be created.

**Screenshots**
Embedded above

**.blend file/ .gltf**
If applicable, a zipped folder containing a .blend file for exporter issues and a .gltf file for importer issues. Please take care to include all resources, such as textures in the .zip file.

**Version**
 - OS: [e.g. macOS, linux] Win 10
 - Blender Version [e.g. 2.82, 2.83, or build date] 2.92

**Additional context**

- If there's anything that seems obvious that I've missed so far, please let me know and I'll remove the issue. Been stuck with this since SU5, and I understand the discord group isn't involved anymore (plus that channel is locked).

**Comment by DevDigitalOfficial** [comment]
im getting the same Error

**Comment by pepperoni505** [comment]
Can you show your addon settings?

**Comment by DevDigitalOfficial** [comment]
> Can you show your addon settings?
![Blender File View 2_11_2022 6_01_05 PM](https://user-images.githubusercontent.com/88412741/153686601-e2b29b86-73d1-4fad-ac07-fb392e25d43d.png)

**Comment by pepperoni505** [comment]
I meant in `Preferences -> Add-ons`

**Comment by DevDigitalOfficial** [comment]
![Blender Preferences 2_11_2022 6_05_14 PM](https://user-images.githubusercontent.com/88412741/153686831-df6a607a-1b66-4c84-935b-253767eb946b.png)
This?

> I meant in `Preferences -> Add-ons`

**Comment by pepperoni505** [comment]
Yeah, that looks right. Are you sure your `texconv.exe` is still at that location? The error message is hard for me to replicate.

**Comment by DevDigitalOfficial** [comment]
> Yeah, that looks right. Are you sure your `texconv.exe` is still at that location? The error message is hard for me to replicate.

yes it is, It only happens when I try to export it for some reason.

**Comment by DevDigitalOfficial** [comment]
![Blender Info Log 2_11_2022 6_22_45 PM](https://user-images.githubusercontent.com/88412741/153687767-b9fc5ffc-3abe-41e4-ab19-4b1b5d7e5513.png)
Now im getting this errror

**Comment by pepperoni505** [comment]
One of your meshes has a very high number of vertices. The exporter is trying to split the mesh into separate primitives, however, there are cases where it is unable to, such as yours. You're going to need to locate any large meshes and either reduce the vert count or split them.

**Comment by DevDigitalOfficial** [comment]
> One of your meshes has a very high number of vertices. The exporter is trying to split the mesh into separate primitives, however, there are cases where it is unable to, such as yours. You're going to need to locate any large meshes and either reduce the vert count or split them.

Alright ill give that a try
